tests/v3_api/test_eks_cluster.py

```diff
@@ -0,0 +1,52 @@
+from .common import *  # NOQA
+
+RANCHER_CLEANUP_CLUSTER = os.environ.get('RANCHER_CLEANUP_CLUSTER', "True")
+ACCESS_KEY = os.environ.get('ACCESS_KEY', "")
+SECRET_KEY = os.environ.get('SECRET_KEY', "")
+
+
+def test_create_eks_cluster():
+
+    if not (ACCESS_KEY and SECRET_KEY):
+        assert (False), "EKS Credentials not provided, cannot create cluster"
+    client = get_admin_client()
+    eksConfig = get_eks_config()
+
+    print("Cluster creation")
+    cluster = client.create_cluster(eksConfig)
+    clusterid = cluster.id
+    print(cluster)
+    print("Cluster list")
+    clusterlist = client.list_cluster(clusterId=clusterid).data
+    cluster = client.reload(clusterlist[0])
+    print(cluster.state)
+    cluster = validate_cluster(client, cluster, check_intermediate_state=True,
+                               skipIngresscheck=True)
+
+    if RANCHER_CLEANUP_CLUSTER == "True":
+        delete_cluster(client, cluster)
+
```

Comment:
Can we add 1 more provisioning test that can take in region and ami as input parameters , so that we can test for different regions and amis.

Reply:
Added code to handle this case in the same test